native-maven-plugin/src/seeding-build/src/test/java/org/graalvm/demo/CalculatorTest.java

```diff
@@ -0,0 +1,38 @@
+package org.graalvm.demo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Do we need these files in this project? 
I guess we only need to specify dependencies and have a basic `HelloWorld` in order for it to pull them?

Reply:
I understand the purpose of the `seeding-build` project, but I don't get why we need to fill that project with code. 
AFAIK we could just have a `HelloWorld.java` and specify that it depends on all of the needed dependencies.

Reply:
Yes, we could also do that, but if you don't have tests, then the Maven surefire plugin wouldn't be downloaded, for example. So it's actually easier to use a "fake" project with real code so that all plugins we use in the similar samples are downloaded. It's easier to reason about IMHO.